noxfile.py

```diff
@@ -20,23 +20,32 @@
 import pathlib
 import shutil
 from pathlib import Path
-from typing import Optional
+from typing import List, Optional
 
 import nox
 
 DEFAULT_PYTHON_VERSION = "3.10"
 CURRENT_DIRECTORY = pathlib.Path(__file__).parent.absolute()
+FLAKE8_VERSION = "flake8==6.1.0"
+BLACK_VERSION = "black[jupyter]==24.8.0"
+ISORT_VERSION = "isort==5.13.2"
```

Comment:
Instead of setting these can you just install the test dependencies from pyproject.toml like `pip install .[test]`?

Reply:
Thanks for showing me this, I've fixed it. Done!